GiasPostcodeSearch/GiasPostcodeSearchService.cs

```diff
@@ -58,13 +58,13 @@ public async Task StartAsync(CancellationToken cancellationToken)
             var subjectsQueryString = GetSubjectsQueryString(schoolDatum);
             if (subjectsQueryString == null)
             {
-                _logger.LogDebug("School {SchoolName} is not valid for this service", schoolDatum?.Name);
+                _logger.LogDebug("School {SchoolName} is not valid for this service", schoolDatum.Name);
             }
             else
             {
                 _logger.LogDebug("Searching for Tuition Partners covering School {SchoolName}", schoolDatum?.Name);
 
-                var requestUri = $"search-results?Data.Postcode={schoolDatum.Postcode}&{subjectsQueryString}";
+                var requestUri = $"search-results?Data.Postcode={schoolDatum!.Postcode}&{subjectsQueryString}";
```

Comment:
I don't think the null-forgiving operator is necessary is it?  My VS says that `schoolDatum` is not null here.